bundle/config/mutator/override_compute.go

```diff
@@ -22,7 +22,12 @@ func (m *overrideCompute) Name() string {
 
 func overrideJobCompute(j *resources.Job, compute string) {
 	for i := range j.Tasks {
-		task := &j.Tasks[i]
+		var task = &j.Tasks[i]
+
+		if task.ForEachTask != nil {
+			task = &task.ForEachTask.Task
```

Comment:
We're no longer overriding fields on original task, is not relevant anymore?

Reply:
If foreach task is set, original task can't specify compute

Reply:
Thanks. This is worth a comment there. 

Is this undocumented? I don't see it on https://docs.databricks.com/api/workspace/jobs/create#tasks-existing_cluster_id

Reply:
It is not well documented in API spec. There will be an API error. There are other tasks that don't allow setting existing_cluster_id as well.